css-typed-om/Overview.bs

```diff
@@ -933,7 +933,9 @@ Shorthand properties and values are not supported.
 
 Issue(159): Spec up ColorValue
 
-Issue(146): Do we need to say that LengthValues must be positive e.g. for border-widths here?
+{{LengthValue}} objects are not range-restricted. Any valid number can be represented by a {{LengthValue}},
```

Comment:
'Any valid combination of primitive lengths can be represented by a {{LengthValue}}'